app/src/components/eventCard/style.module.scss

```diff
@@ -1,5 +1,5 @@
-$cont-width:250px;
```

Comment:
plz use the variable in a separate file. see task F29
https://github.com/i-barysiuk/bstu-curators/pull/83/files